libs/dal/src/repositories/notification/notification.entity.ts

```diff
@@ -37,9 +37,7 @@ export class NotificationEntity {
   createdAt?: string;
   updatedAt?: string;
   expireAt?: string;
-
-  bridge?: any;
-
+  tags: string[];
```

Comment:
We don't need to store Bridge data on the Notification entity, only the `tags` associated with the Notification from the Workflow.